[2024-10-24T20:58:08.485+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-24T20:58:35.271+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: SetupETLtables.createTable__1 manual__2024-10-24T20:57:23.389746+00:00 [queued]>
[2024-10-24T20:58:35.607+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: SetupETLtables.createTable__1 manual__2024-10-24T20:57:23.389746+00:00 [queued]>
[2024-10-24T20:58:35.619+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-24T20:58:48.888+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): createTable__1> on 2024-10-24 20:57:23.389746+00:00
[2024-10-24T20:58:49.071+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'SetupETLtables', 'createTable__1', 'manual__2024-10-24T20:57:23.389746+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/etl_table_snowflake.py', '--cfg-path', '/tmp/tmpw3j_wz_d']
[2024-10-24T20:58:51.169+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=295) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-24T20:58:51.462+0000] {standard_task_runner.py:91} INFO - Job 41: Subtask createTable__1
[2024-10-24T20:58:54.878+0000] {standard_task_runner.py:63} INFO - Started process 335 to run task
[2024-10-24T21:00:30.447+0000] {task_command.py:426} INFO - Running <TaskInstance: SetupETLtables.createTable__1 manual__2024-10-24T20:57:23.389746+00:00 [running]> on host 599a8e9540bd
[2024-10-24T21:00:48.980+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='SetupETLtables' AIRFLOW_CTX_TASK_ID='createTable__1' AIRFLOW_CTX_EXECUTION_DATE='2024-10-24T20:57:23.389746+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-24T20:57:23.389746+00:00'
[2024-10-24T21:00:49.018+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-24T21:00:49.109+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn' for task execution.
[2024-10-24T21:00:49.122+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-6.10.4-linuxkit-aarch64-with-glibc2.36
[2024-10-24T21:00:49.144+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-10-24T21:00:50.478+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2024-10-24T21:00:50.591+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2024-10-24T21:00:50.592+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-24T21:00:50.595+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-24T21:00:50.641+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=SetupETLtables, task_id=createTable__1, run_id=manual__2024-10-24T20:57:23.389746+00:00, execution_date=20241024T205723, start_date=20241024T205835, end_date=20241024T210050
[2024-10-24T21:00:50.702+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-24T21:00:50.752+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-24T21:00:50.755+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
